aws-lc-rs/src/error.rs

```diff
@@ -7,7 +7,9 @@
 
 extern crate std;
 
-use std::num::TryFromIntError;
+use core::num::TryFromIntError;
+// The Error trait is not in core: https://github.com/rust-lang/rust/issues/103765
+use std::error::Error;
```

Comment:
`core::error:Error` is [not yet stable](https://github.com/rust-lang/rust/issues/103765)